content/60.methods.md

```diff
@@ -12,26 +12,36 @@ Below, we will describe the processes used to synthesize the literature.
-Similarly, Figure @fig:ebm-trials was generated based on data from the University of Oxford Evidence-Based Medicine (EBM) Data Lab's COVID-19 TrialsTracker [@doi:10.5281/zenodo.3732709].
-In both cases, frequency data was plotted using MatPlotLib in Python.
-Figure @fig:ebm-map was generated using the countries associated with the trials listed in the EBM's COVID-19 TrialsTracker, converting the country names to 3-letter ISO codes using pycountry or manual adjustment when necessary, and visualizing the geographic distribution of trial recruitment using geopandas. 
-Current version information for packages and software is available in https://github.com/greenelab/covid19-review/blob/external-resources/environment.yml.
-
-<!-- Add in RNA-Seq analysis-->
+The combination of Manubot and GitHub Actions also made it possible to dynamically update information such as statistics and visualizations in the manuscript.
+Data about worldwide cases and deaths from the COVID-19 Data Repository by the Center for Systems Science and Engineering at Johns Hopkins University [@https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_time_series] were read using a Python script to generate Figure @fig:csse-deaths.
+Similarly, Figure @fig:ebm-trials and clinical trials statistics were generated based on data from the University of Oxford Evidence-Based Medicine Data Lab's COVID-19 TrialsTracker [@doi:10.5281/zenodo.3732709].
+In both cases, frequency data were plotted using Matplotlib [@doi:10.1109/MCSE.2007.55] in Python.
+Figure @fig:ebm-map was generated using the countries associated with the trials listed in the COVID-19 TrialsTracker, converting the country names to 3-letter ISO codes using pycountry or manual adjustment when necessary, and visualizing the geographic distribution of trial recruitment using geopandas.
+<!-- Is this figure going to be included? #673 -->
+
+GitHub Actions runs a nightly workflow to update these external data and regenerate the statistics and figures for the manuscript.
+The workflow uses the GitHub API to detect and save the latest commit of the external data sources, which are both GitHub repositories.
+It then downloads versioned data from that snapshot of the external repositories and runs bash and Python scripts to calculate the desired statistics and produce the summary figures.
+The statistics are stored in JSON files that are accessed by Manubot to populate the values of placeholder template variables dynamically every time the manuscript is built.
+For instance, the template variable {% raw %}`{{ebm_trials_results}}`{% endraw %} in the manuscript is replaced by the actual number of clinical trials with results, {{ebm_trials_results}}.
```

Comment:
Ahh - I see you did it here - very nice!

Reply:
Would it be better if I move the example earlier?

Reply:
I don't have strong feelings - I made the first comment when I wanted to see it but then this came along with everything I was hoping to find.

Reply:
Sounds good.  I'm going to merge given your approval and can make new updates if @rando2 has any comments later.